src/_scss/components/_card.scss

```diff
@@ -3,7 +3,69 @@
   background-color: white;
   border-radius: 6px;
   margin: 0 0 $global-margin * 4;
-  padding: $global-padding * 2;
+  padding: $global-padding * 3;
+  @media (min-width: $medium-screen) {
+    display: flex;
+    align-items: center;
+    padding: $global-margin * 5;
+  }
   min-height: 250px;
   box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
-}
\ No newline at end of file
+
+  .card__icon-wrapper {
+    width: 35px;
+    height: 35px;
+    background: #1b1b1b;
+    text-align: center;
+    border-radius: 4px;
+    padding: $global-padding;
+    @media (max-width: $medium-screen) {
+      margin-bottom: rem(16);
+    }
+  }
+  .card__heading-wrapper {
+    font-size: $h4-font-size;
+    font-weight: $font-semibold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.25;
+    letter-spacing: normal;
+    color: #1b1b1b;
+    @media (min-width: $medium-screen) {
+      margin-left: $global-margin * 2;
+      padding-bottom: 2px;
+    }
+
+    p {
+      margin: 0;
+    }
+  }
+  
+  .card__content-wrapper {
+    font-size: $base-font-size;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.5;
+    letter-spacing: normal;
+    color: #555;
```

Comment:
Could you use $color-base from _variables.scss here instead of the hex code #555. There are a few other places below too